app/forms/schools/coaches/create_form.rb

```diff
@@ -31,7 +54,8 @@ def faculty_params
           connect_link: connect_link,
           public: public,
           image: image,
-          affiliation: affiliation
+          affiliation: affiliation,
+          archived_at: archived_at == "true" ? Time.zone.now : nil
```

Comment:
Why should we allow the creation of archived users? 🤔

Reply:
IDK, should we?